lib/Models/sendFeedback.ts

```diff
@@ -0,0 +1,105 @@
+import i18next from "i18next";
+import isDefined from "../Core/isDefined";
+import loadWithXhr from "../Core/loadWithXhr";
+import TerriaError from "../Core/TerriaError";
+import {
+  buildShareLink,
+  buildShortShareLink,
+  canShorten
+} from "../ReactViews/Map/Panels/SharePanel/BuildShareLink";
+import Terria from "./Terria";
+
+export default function sendFeedback(options: {
+  terria: Terria;
+  title?: string;
+  name: string;
+  email: string;
+  sendShareURL: boolean;
+  comment: string;
+  additionalParameters?: Record<string, string | undefined>;
+}) {
+  if (!isDefined(options) || !isDefined(options.terria)) {
+    throw TerriaError.from("options.terria is required.");
+  }
+
+  var terria = options.terria;
```

Comment:
could be a const, but its such a nitpick, i'm not fussed

Reply:
👍